core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -130,11 +135,18 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
                     }).pipe(
                         tap({
                             next: ({ pageData, licenseData }) => {
+                                const isHeadlessPage = !!params.clientHost;
                                 this.setState({
                                     clientHost: params.clientHost,
                                     editor: pageData,
+                                    vtlIframeContent: isHeadlessPage
+                                        ? null
+                                        : createIframeUrlByRenderedPage(pageData.page.rendered),
                                     isEnterpriseLicense: licenseData,
-                                    editorState: EDITOR_STATE.LOADING
+                                    //This to stop the progress bar. Testing yet
+                                    editorState: isHeadlessPage
+                                        ? EDITOR_STATE.LOADING
+                                        : EDITOR_STATE.LOADED
```

Comment:
Not sure... probably for now, we will have to do extra like listen the loading maybe later.